carta/image.py

```diff
@@ -477,17 +477,24 @@ def set_scaling(self, scaling, alpha=None, gamma=None, min=None, max=None):
             The alpha value (only applicable to ``LOG`` and ``POWER`` scaling types).
         gamma : {2}
             The gamma value (only applicable to the ``GAMMA`` scaling type).
-        min : {3}
-            The minimum of the scale. Only used if both *min* and *max* are set.
-        max : {4}
-            The maximum of the scale. Only used if both *min* and *max* are set.
+        clip : {3}
+            The scale clip. In percentage. Can not be used witn *min* and *max*.
+        min : {4}
+            The minimum of the scale. Only used if both *min* and *max* are set. Can not be used with *clip*.
+        max : {5}
+            The maximum of the scale. Only used if both *min* and *max* are set. Can not be used with *clip*.
```

Comment:
These descriptions should be changed to:

 `The clip percentage. If this is set, *min* and *max* are ignored.`

`Custom clip minimum. Only used if both *min* and *max* are set. Ignored if *clip* is set.`

`Custom clip maximum. Only used if both *min* and *max* are set. Ignored if *clip* is set.`

Reply:
Done in 7b81303.

To avoid confusion, I also replace the argument from `clip` to `rank` to unify the name of the argument with `set_percentile_rank`. The description of `rank` is thus modified the replacement:

```
rank : {3}
    The percentile rank. If this is set, *min* and *max* are ignored, and will be calculated automatically.
min : {4}
    Custom clip minimum. Only used if both *min* and *max* are set. Ignored if *rank* is set.
max : {5}
    Custom clip maximum. Only used if both *min* and *max* are set. Ignored if *rank* is set.
```